src/mixins/name-request-mixin.ts

```diff
@@ -17,8 +17,7 @@ export default class NameRequestMixin extends Mixins(DateMixin) {
   generateNameRequestState (nr: any, filingId: number): NameRequestIF {
     return {
       nrNumber: nr.nrNum,
-      // TODO: Update entityType to use nr.requestTypeCd when namex supports our entity types
-      entityType: CorpTypeCd.BENEFIT_COMPANY,
```

Comment:
I'm not sure how this code worked for CR/UL/CC previously ... Hmm, everything was not a BEN until now, was it?

Reply:
Oh, I guess the previous code worked because we got the entity type from the draft IA, not from the NR (and we didn't check if they matched).